src/subdomains/supporting/payin/entities/crypto-input.entity.ts

```diff
@@ -259,4 +278,12 @@ export class CryptoInput extends IEntity {
 
     return this;
   }
+
+  get isLightningInput(): boolean {
+    return this.asset.blockchain === Blockchain.LIGHTNING;
+  }
+
+  get amlReason(): AmlReason | 'staking' {
+    return this.route instanceof Staking ? 'staking' : AmlReason.ASSET_CURRENTLY_NOT_AVAILABLE;
+  }
```

Comment:
I think we can add STAKING_DISCONTINUED (or similar) to AML reasons?

Reply:
Hm yeah thought about that too 🤔 
It will be not really used as amlReason, so in fact it's not an amlReason but yeah we could nevertheless add it as amlReason 🤔